Doc/library/subprocess.rst

```diff
@@ -1538,8 +1538,8 @@ handling consistency are valid for these functions.
       as it did in Python 3.3.3 and earlier.  exitcode has the same value as
       :attr:`~Popen.returncode`.
 
-   .. versionadded:: 3.11
-      Added *encoding* and *errors* arguments.
+   .. versionchanged:: 3.11
+      Added the *encoding* and *errors* arguments.
```

Comment:
```suggestion
      Added the *encoding* and *errors* parameters.
```

If we're changing it elsewhere to be consistent, since we're touching this line anyway?